apollo-portal/src/test/java/com/ctrip/framework/apollo/openapi/v1/controller/NamespaceControllerTest.java

```diff
@@ -34,6 +37,10 @@
  */
 @ActiveProfiles("skipAuthorization")
 public class NamespaceControllerTest extends AbstractControllerTest {
+
+  static final HttpHeaders HTTP_HEADERS_WITH_TOKEN = new HttpHeaders() {{
```

Comment:
Looks like the `consumerAuthUtil` mock isn't cleared which caused you add this logic, which is unnecessary. So it's better to fix the `consumerAuthUtil` mock issue instead of adding this.

Reply:
I will restore this test case code and improve the `consumerAuthUtil` mock